File: src/main/java/no/nav/boot/conditionals/OnSBSCondition.java
package no.nav.boot.conditionals;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnSBSCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return Cluster.sbsClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnSBSCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.sbsClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnSBSCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = sbsClusters()
}